data/fields/contact/instagram.json

```
@@ -0,0 +1,6 @@
+{
+    "key": "contact:instagram",
+    "type": "url",
+    "label": "Instagram account name (also used on Threads) or full Instagram URL",
+    "placeholder": "account_name"
```

Full URLs seem to be slightly more popular at first glance (see [taginfo](https://taginfo.openstreetmap.org/keys/?key=contact%3Ainstagram#values)) and are probably easier to copy-paste, so maybe use something like this for the placeholder:

```suggestion
    "placeholder": "https://www.instagram.com/LocalBusiness"
```

---

I _personally_ prefer using the account name because I've never seen a business advertise their Instagram with the URL, but I've updated the PR to use a URL. I took the liberty of using https://www.instagram.com/openstreetmap/ which seems like a legitimate Instagram account.